tor/__init__.py

```diff
@@ -1,49 +1 @@
-import bugsnag
-from addict import Dict
-
 __version__ = '2.6.14'
```

Comment:
Heh, I was avoiding touching a version increment until after merging a PR. It's especially useful if there are multiple PRs in a single release. Didn't want to presume.